sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdServiceEndpoint.java

```diff
@@ -111,12 +111,12 @@ private RntbdServiceEndpoint(
 
     @Override
     public int channelsAcquired() {
-        return this.channelPool.channelsAcquired();
```

Comment:
Both metrics below are not consistent right?

Reply:
ACK - renamed to *metric and added explanation